### DISCUSSION TITLE
Too many requests sent in a short time period

### DISCUSSION THREAD

**Discussion by bwynants** [bimmerconnected/bimmer_connected#635] [General]
I use the BMW Connected Drive to make sure the charging follows the power generated by my solar panels.

After sending  request (charging limit)  i put in a delay so that i do not have too many requests.... however sometimes the automation stops the log file reports  
Remote service failed with state 'ExecutionState.ERROR'. Response: {'eventStatus': 'ERROR', 'errorDetails': {'title': 'Multiple requests sent to vehicle', 'description': 'Too many requests sent in a short time period. Please wait for couple of minutes to send request again.', 'presentationType': 'DIALOG', 'iconId': None, 'isRetriable': True, 'errorDetails': 'REFUSED_PLAY_PROTECTION'}}

after a while it picks up again.

However I want to handle the 'Too many requests sent in a short time period' how do i catch this error in the automation so that i can put in a delay of 5 minutes before sending another request to the service?

**Comment by rikroe** [comment]
Unfortunately this is currently not possible within an automation. See https://github.com/home-assistant/architecture/discussions/845.
